boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -26,8 +26,16 @@
 				reg = <0x800 DT_SIZE_K(2)>;
 			};
 
-			cpuapp_data: memory@1000 {
-				reg = <0x1000 DT_SIZE_K(256)>;
+			cpuapp_cpusec_misc_shm: memory@1000 {
```

Comment:
@hubertmis should we disable this region by default?

Reply:
I understand this region is needed for IPC with SecDom. Let's keep it enabled by default, similarly to `cpusec_cpuapp_ipc_shm`.